### ISSUE TITLE
How can I collect two-day data with Scouter API?

### ISSUE THREAD

**Issue by danielelic** [scouter-project/scouter#502]
I would like to extract with a single request (`/v1/xlog-data/{yyyymmdd}`) all the data at between 11:00 pm and 01:00 am. Is it possible?

**Comment by gunlee01** [comment]
No. And the XLog API allows you to view up to 10,000 logs logs at a time.
The response includes `lastTxid` and `lastXLogTime`, and you can add this value to the next request to retrieve the next 10,000 logs.

* Example: `/v1/xlog-data/ {yyyymmdd}?startTimeMills={value}&endTimeMillis={value}&objHashes={value}&lastTxid={value}&lastXLogTime={value}`

> xlog api(not xlog-data api) is more efficient for bulk load. But you need to get dictionary decoding values by calling dictionary api.

**Comment by danielelic** [comment]
Thank you! Will the date constraint (`{yyyymmdd}`) be removed in the next xlog-data API?

**Comment by gunlee01** [comment]
Using objHash as a non-numeric character, using the numeric form of the response JSON format as a number rather than a character, and removing the date condition is a consideration in the v2 API. However, specific plans and schedules for v2 API are not fixed.
